modules/exploits/linux/local/dirtycow.rb

```diff
@@ -0,0 +1,176 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+require 'msf/core/exploit/local/linux_kernel'
+require 'msf/core/exploit/local/linux'
+require 'msf/core/exploit/exe'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GreatRanking
+
+  include Msf::Exploit::EXE
+  include Msf::Post::File
+  include Msf::Exploit::FileDropper
+  include Msf::Exploit::Local::Linux
+
+  def initialize(info={})
+    super(update_info(info, {
+      'Name' => 'Linux Kernel DirtyCow Local Privilege Escation',
+      'Description' => %q{
```

Comment:
no mention of which kernels are vulnerable

Reply:
this is scary , our college library uses some modified version of ubuntu called koha i think it is using this old kernal